go/stellar/stellar.go

```diff
@@ -77,7 +104,7 @@ func CreateWalletSoft(ctx context.Context, g *libkb.GlobalContext) {
 		err = fmt.Errorf("yielding to guard")
 		return
 	}
-	_, err = CreateWalletGated(ctx, g)
+	_, _, err = CreateWalletGated(ctx, g)
```

Comment:
This `err` is ignored...check it?  log it?

Reply:
It's logged by the trace

Reply:
ok